quickstarts/manage-data/automate-data-movement-and-transformation-with-airflow-airbyte-and-dbt-in-teradata-vantage.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 5.5
+title: Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+author: Vidhan Bhonsle
+email: vidhan.bhonsle@teradata.com
+page_last_update: October 4th, 2024
+description: Using Airflow to Orchestrate Airbyte and dbt for Teradata
+keywords: [data warehouses, airflow, teradata, vantage, orchestration, object storage, airbyte, enterprise analytics, elt, dbt.]
+---
+
+import ClearscapeDocsNote from '../_partials/vantage_clearscape_analytics.mdx'
+import CommunityLink from '../_partials/community_link.mdx'
+import ProfileTabs from '../_partials/tabsDBTProfiles.mdx'
+
+# Automate Data Movement and Transformation with Airflow, Airbyte, and dbt in Teradata Vantage
+
+## Overview
+
+This quickstart demonstrates how to leverage Apache Airflow as orchestration tool to automate the movement of data to Teradata using Airbyte. Once the data is in Teradata, trigger dbt to perform transformations, ensuring the data is clean, reliable and ready for analysis.
+
+## Prerequisites
+
+* Access to a Teradata Vantage instance.
+
+    <ClearscapeDocsNote />
+
+* [Python](https://www.python.org/downloads/) **3.10** or later installed.
+
+* [Docker and Docker Compose (Docker Desktop)](https://docs.docker.com/get-started/get-docker/)
+
+* [Airbyte command line tool](https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart)
+
+## Demo project setup
+
+1. Clone the tutorial repository:
+```bash
+git clone https://github.com/Teradata/airflow-demos.git
+```
+
+2. Navigate to the directory:
+```bash
+cd airflow-demos/airbyte_dbt_airflow_teradata
+```
+Open the code in your preferred IDE.
+
+
+## Setting up Teradata instance
+Follow the instructions from [Getting started with ClearScape Analytics Experience](https://developers.teradata.com/quickstarts/get-access-to-vantage/clearscape-analytics-experience/getting-started-with-csae/) and acquire following:
+
+*  Host
+* Username
+* Password
+
+Jot it down, it will be needed in next step.
+
+## Airbyte setup
+
+1. After you install Airbyte OSS locally using `abctl`, generate and copy the credentials:
+
+```bash
+abctl local credentials
+```
+
+:::note 
+At the moment of writing this quickstart, Airbyte installed with `abctl` performs unauthenticated pulls even if you pass in Docker credentials: https://github.com/airbytehq/airbyte/issues/46309. Since Docker limits unauthenticated pulls you may encounter `429 Too Many Requests` error. If you run into this error, launch Airbyte using `run-ab-platform.sh` scruot as described in [the Airbyte quickstart](https://developers.teradata.com/quickstarts/manage-data/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage/#airbyte-open-source).
```

Comment:
```suggestion
At the moment of writing this quickstart, Airbyte installed with `abctl` performs unauthenticated pulls even if you pass in Docker credentials: https://github.com/airbytehq/airbyte/issues/46309. Since Docker limits unauthenticated pulls you may encounter `429 Too Many Requests` error. If you run into this error, launch Airbyte using `run-ab-platform.sh` script as described in [the Airbyte quickstart](https://developers.teradata.com/quickstarts/manage-data/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage/#airbyte-open-source).
```